docs/source/sctool/partials/sctool_repair.yaml

```diff
@@ -108,6 +110,8 @@ options:
     The effective parallelism depends on a keyspace replication factor (RF) and the number of nodes.
     The formula to calculate is is as follows: number of nodes / RF.
     For example, the maximum parallelism for a 6 node cluster with RF=3 is 2.
+    Setting parallel to a value greater than max supported results in trimming it internally.
```

Comment:
See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.